packages/expo-image-picker/ios/ImagePickerOptions.swift

```diff
@@ -13,6 +13,9 @@ internal struct ImagePickerOptions: Record {
   @Field
   var allowsEditing: Bool = false
 
+  @Field
+  var useLivePhotos: Bool = false
```

Comment:
Instead of introducing another field, could we make `mediaTypes` an array and add `livePhotos` case to `MediaType`?